app/Http/Controllers/CompanyController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Models\Company;
+use Illuminate\Http\Request;
+
+class CompanyController extends Controller
+{
+    public function index(Request $request)
```

Comment:
questa funzione NON si deve chiamare index per index si intende l'index di company in questo caso tornda un sub dataset di companu chiamare la funzione formJson